python/lsst/ts/dsm/utils.py

```diff
@@ -5,7 +5,7 @@
 from astropy.time import Time, TimeDelta
 import numpy as np
 
-from lsst.ts import salobj
+from lsst.ts import utils
```

Comment:
Since you are in module utils it might be clearer to use:
```
from lsst.ts.utils import tai_from_utc
```
or
```
import lsst.ts.utils as tsUtils (matching what you do indsm_csc.py)
```
it would enhance readability to use the same convention in both of your library modules

Reply:
Done. Used second suggestion.